### DISCUSSION TITLE
How to remove keywords like "via" and "on"

### DISCUSSION THREAD

**Discussion by behzadsh** [starship/starship#1318] [General]
Is there any way to remove keywords before the username and git branch name?

**Comment by chipbuster** [comment]
tl;dr This will become available after the config string update, which is currently in the process of being merged. Unfortunately, the merge is going slower than I'd like right now for various reasons, but once it's done, this should be possible.

If you'd like to track the progress of this, you can look at #1057 for the tracking issue, and [the milestone](https://github.com/starship/starship/milestone/3) for a rough idea of how close we are.

**Comment by pbnj** [comment]
Update since the [last comment](https://github.com/starship/starship/discussions/1318#discussioncomment-25730), use a preset:

```shell
$ starship preset bracketed-segments
  
[aws]
format = '\[[$symbol($profile)(\($region\))(\[$duration\])]($style)\]'
...
```

Now you can copy/paste select segments that you want or dump it all into your config file with `starship preset bracketed-segments >> /path/to/your/starship.toml`
